src/interfaces/Attribute.ts

```diff
@@ -0,0 +1,20 @@
+export interface Attribute {
```

Comment:
I think this should extend Orbit's type. Also, `name`, `description` etc., are specific to `Product` model.